pennylane/gradients/parameter_shift.py

```diff
@@ -73,42 +73,40 @@ def _get_operation_recipe(tape, t_idx, shifts):
 
     This function performs multiple attempts to obtain the recipe:
 
+    - If the operation has a custom ``grad_recipe`` defined, it is used.

```

Comment:
```suggestion
    - If the operation has a custom :attr:`~.grad_recipe` defined, it is used.
```

Reply:
@dwierichs I only realized after leaving all these comments that this is a private function 🤦‍♂️ 

Having said that, I think this is _very_ useful information, and it would be worth copying this (as well as the cross-ref suggestions) into the docstring for [`qml.gradients.param_shift`](https://pennylane--2227.org.readthedocs.build/en/2227/code/api/pennylane.gradients.param_shift.html) so that it is visible :)